Sample-01/tests/pages/api/shows.test.js

```diff
@@ -2,7 +2,7 @@
  * @jest-environment node
```

Comment:
Should we update the dir structure of the tests?

Reply:
Done [e2220b5](https://github.com/auth0-samples/auth0-nextjs-samples/pull/131/commits/e2220b51cb725f3459b2e601906a677500d9a92e)